src/arrayMethodSort.js

```diff
@@ -4,8 +4,33 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareStrings = (a, b) => {
+    return (String(a) > String(b))
+      ? 1
+      : (String(a) < String(b))
+        ? -1
+        : 0;
```

Comment:
I'm not sure, but it should probably work.

```suggestion
    return (String(a) < String(b))
      ? -1
      : (String(a) > String(b))
```

Reply:
But personally, I would rewrite it to ifs :))